### PR TITLE
fine tuning prompts and inserting answers to firebase

### PR DESCRIPTION
**Pull request by samridhijha**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -2,41 +2,110 @@
...
+                      type: object
+                      properties:
+                        answers:
+                          type: object
+                          properties:
+                            answers:
+                              type: object
+                              properties:
+                                symptoms:
+                                  type: string
```
Comment (blatantlyAxy) [inline app.py:41]: do we need this structure here as part of the code? If not, please remove it and add it to a sample.txt file.

Comment (SriramMedisetti) [reply app.py:41]: yes we need this , based on his only - end point will allow the json fields

Comment (blatantlyAxy) [reply app.py:41]: can this be moved into another file instead of here and then pulled in when needed?

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,323 @@
+alabaster @ file:///home/ktietz/src/ci/alabaster_1611921544520/work
```
Comment (jajatikr) [inline requirements.txt:1]: I think the requirements.txt can be simplified. It looks like packages from projects has crept into this file.

**Pull request merged**